BusBoard/Models/BusArrivalPrediction.cs

```diff
@@ -1,6 +1,6 @@
 namespace BusBoard.Models;
 
-public class Prediction
+public record BusArrivalPrediction

```

Comment:
you could update this to be a constructor declaration
```suggestion
public record BusArrivalPrediction(
    string LineName,
    public DateTime);

```